dsgrid_project/build_registry/README.md

```diff
@@ -0,0 +1,59 @@
+# Build StandardScenarios registry
+The `build_registry.sh` script in this directory builds a dsgrid registry containing all
+StandardScenarios datasets and derived datasets. dsgrid developers need to keep the scripts updated
+as new derived datasets are added.
+
+## Dependencies
+The scripts rely on Apache Spark cluster creation scripts in
+https://github.com/NREL/HPC/tree/master/applications/spark. You must clone it locally.
+```
+$ git clone https://github.com/NREL/HPC
+```
+
+## Preparation
+The script will generate a large amount of data (currently 8-9 TiB). Run the script from
+`/scratch/$USER`. Ensure that the data is correct before you copy it to `/projects/dsgrid` or AWS.
+
+The rest of these instructions assume that you are working from an empty directory in
+`/scratch/$USER`.
+
+Edit `${HOME}/.dsgrid.json5` to point to the correct database and ensure that ``offline`` is true.
+If you don't have the file, run this command with correct values:
+```
+$ dsgrid config create --database-name TEXT --database-url TEXT --offline
+```
```

Comment:
I did edit this text and consider it fixed. The database name and url fields should now be clear after the dsgrid fixes I pushed.

Reply:
Got it. I was definitely confused by the long URL in the dsgrid CLI help.

Otherwise, I'm a little bit torn between wanting some mention of the arango database in the dependencies and appreciating that our guidance for that is still in considerable flux.